multi-currency/src/MultiCurrency.php

```diff
@@ -207,6 +207,19 @@ public function __construct( array $gateway_context, MultiCurrencyApiClientInter
 		$this->currency_switcher_block = new CurrencySwitcherBlock( $this, $this->compatibility );
 	}
 
+	/**
+	 * Backwards compatibility for the old `instance()` static method.
+	 *
+	 * We need to use this as some plugins still call `MultiCurrency::instance()` directly.
+	 *
+	 * @return null|MultiCurrency - Main instance.
+	 */
+	public static function instance() {
+		if ( function_exists( 'WC_Payments_Multi_Currency' ) ) {
```

Comment:
Nit: We discussed this through Slack, but to reiterate, we may need to consider adding a deprecation message. We agreed that we need to discuss how to proceed further, so for now, we can leave it as-is.